aten/src/ATen/mps/MPSAllocator.mm

```diff
@@ -567,20 +583,12 @@ explicit MPSAllocator(uint32_t Usage) :
   {
     if (_getAllocImpl().getDebugVerbosity()) {
       if (!(m_usage & HeapAllocator::UsageFlags::SHARED) || m_has_unified_memory) {
-        const size_t high_watermark_limit = _getAllocImpl().getHighWatermarkLimit();
-        const size_t low_watermark_limit  = _getAllocImpl().getLowWatermarkLimit();
         std::cerr << "Initializing "
                   << ((m_usage & HeapAllocator::UsageFlags::SHARED) ? "shared" : "private")
                   << " heap allocator on "
                   << (m_has_unified_memory ? "unified" : "discrete")
                   << " device memory of size "
-                  << _getAllocImpl().Device().recommendedMaxWorkingSetSize / 1048576UL << " MB"
-                  << " (max allowed: "
-                  << (high_watermark_limit == std::numeric_limits<size_t>::max() ? "unlimited" :
-                     (to_string(high_watermark_limit / 1048576UL) + " MB"))
-                  << ", low watermark: "
-                  << (low_watermark_limit == std::numeric_limits<size_t>::max() ? "unlimited" :
-                     (to_string(low_watermark_limit / 1048576UL) + " MB"))  << ")\n";
+                  << _getAllocImpl().Device().recommendedMaxWorkingSetSize / 1048576UL << " MB\n";
```

Comment:
Better to make "1048576UL" readable than a magic number ?

Reply:
Updated the PR